omnigibson/objects/object_base.py

```diff
@@ -112,13 +112,17 @@ def __init__(
         self._init_info["args"]["name"] = self.name
         self._init_info["args"]["uuid"] = self.uuid
 
-    def load(self):
-        # Run super method ONLY if we're not loaded yet
-        if self.loaded:
-            prim = self._prim
-        else:
-            prim = super().load()
-            log.info(f"Loaded {self.name} at {self.prim_path}")
+    def prebuild(self, stage):
```

Comment:
should this be applied to prims also? or is this specifically OG-object centric?

Reply:
@cgokmen

Reply:
Object-centric mainly for dataset objects for now